docs-source/content/faq/resource-settings.md

```diff
@@ -0,0 +1,162 @@
+
+```
+  spec:
+    resources:
+      env:
+      - name: USER_MEM_ARGS
+        value: "--XX:MinRAMPercentage=25.0 --XX:MaxRAMPercentage=50.0 -Djava.security.egd=file:/dev/./urandom"
+```
+
+Additionally there's also a node-manager process that's running in the same container as the WebLogic Server which has its own heap and off-heap requirements. Its heap is tuned by using `-Xms` and `-Xmx` in the `NODEMGR_MEM_ARGS` environment variable. Oracle recommends setting the node manager heap memory to fixed sizes, instead of percentages, where [the default tuning]({{< relref "/userguide/managing-domains/domain-resource#jvm-memory-and-java-option-environment-variables" >}}) is usually sufficient.
+
+{{% notice warning %}}
+If you set `USER_MEM_ARGS` or `NODEMGR_MEM_ARGS` in your domain resource, then it is usually recommended to include `-Djava.security.egd=file:/dev/./urandom` in order to speedup boot times on systems with low entropy. This setting is included in the respective defaults for these two environment variables.
+q
+{{% /notice %}}
+
+In some cases, you might only want to configure memory resource requests but not configure memory resource limits. In such scenarios, you can use the traditional fixed heap size settings (`-Xms` and `-Xmx`) in your WebLogic Server `USER_MEM_ARGS` instead of the percentage settings (`-XX:MinRAMPercentage` and `-XX:MaxRAMPercentage`).
+
+### CPU resource considerations
+
+It's important to set both a CPU request and a limit for WebLogic Server pods. This ensures that all WebLogic server pods have enough CPU resources, and, as discussed earlier, if the request and limit are set to the same value, then they get a `guaranteed` QoS. A `guaranteed` QoS ensures the pods are handled with a higher priority during scheduling and so are the least likely to be evicted.
+
+If a CPU request and limit are _not_ configured for a WebLogic Server pod:
+- The pod can end up using all CPU resources available on its node and starve other containers from using shareable CPU cycles. 
+
+- The WebLogic server JVM may choose an unsuitable garbage collection (GC) strategy.
+
+- A WebLogic Server self-tuning work-manager may incorrectly optimize the number of threads it allocates for the default thread pool. 
+
+It's also important to keep in mind that if you set a value of CPU core count that's larger than core count of your biggest node, then the pod will never be scheduled. Let's say you have a pod that needs 4 cores but you have a kubernetes cluster that's comprised of 2 core VMs. In this case, your pod will never be scheduled.  
```

Comment:
It would be more helpful to show an example of the error when a Pod is never scheduled (it will be stuck in "Pending").

Reply:
Added an example showing pod in Pending state.